src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/AddPermissionsAndEmployerController.cs

```diff
@@ -2,21 +2,26 @@
 using FluentValidation.AspNetCore;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Provider.PR.Domain.Interfaces;
+using SFA.DAS.Provider.PR.Domain.OuterApi.Requests.Commands;
 using SFA.DAS.Provider.PR.Web.Authorization;
 using SFA.DAS.Provider.PR.Web.Constants;
+using SFA.DAS.Provider.PR.Web.Extensions;
 using SFA.DAS.Provider.PR.Web.Infrastructure;
 using SFA.DAS.Provider.PR.Web.Infrastructure.Services;
 using SFA.DAS.Provider.PR.Web.Models.AddEmployer;
 using SFA.DAS.Provider.PR.Web.Models.Session;
+using SFA.DAS.Provider.PR.Web.Services;
 
 namespace SFA.DAS.Provider.PR.Web.Controllers.AddEmployer;
 
 [Authorize(Policy = nameof(PolicyNames.HasContributorOrAbovePermission))]
 
 [Route("/{ukprn}/addEmployer/addPermissions", Name = RouteNames.AddPermissionsAndEmployer)]
```

Comment:
The user action does not add permission, it requests it. I think the route `/ukprn/addemployer` is sufficient as it implicitly then results in requesting permissions.

Reply:
ok